### DISCUSSION TITLE
Raw HTML module

### DISCUSSION THREAD

**Discussion by iorbita** [templaza/astroid-framework#353] [Ideas]
Hi,
It would be great if in the new version of Astroid we could insert a "raw HTML" module in the layout section (Easy Layout component uses this concept), a module with no pre-written HTML code, so no columns, no rows…nothing! but a simple field where you can enter your own HTML code. Sometimes you don't need lots of nested div tags but just a few lines of HTML code.

Thanks,
Lorenzo

**Comment by Chacapamac** [comment]
I guess this can be useful in certain conditions.
